openshift_metrics/merge.py

```diff
@@ -210,15 +227,24 @@ def main():
     else:
         pod_report_file = f"Pod NERC OpenShift {report_month}.csv"
 
-    report_start_date = datetime.strptime(report_start_date, "%Y-%m-%d").replace(
+    report_start_date_dt = datetime.strptime(report_start_date, "%Y-%m-%d").replace(
+        tzinfo=UTC
+    )
+    report_end_date_dt = datetime.strptime(report_end_date, "%Y-%m-%d").replace(
```

Comment:
If you want to indicate that `report_start_date` and `report_end_date` are `datetime` objects, I would suggest using type annotation instead, which I think would be simpler to interpret than having to rename references and assignements.

```suggestion
    report_start_date : datetime = datetime.strptime(report_start_date, "%Y-%m-%d").replace(
        tzinfo=UTC
    )
    report_end_date : datetime = datetime.strptime(report_end_date, "%Y-%m-%d").replace(
        tzinfo=UTC
    )
```

Is there another reason why you renamed these variables?